wordpress-mobile/WordPress-iOS
# Add UI tests for critical login flows

Part of #11013 

Adds tests for the following login flows:

* Log in to WordPress.com using site address option.
* Log in to self-hosted site (using site address option) and log out again. This test is currently disabled as there's a crash right after completing the test steps (see #11094).
* Start the login flow with the magic link option (ending with the step to request the link). We can consider options for expanding this test once our planned network mocking is in place.

This also cleans up the page models for the login pages, so all strings are defined separately and can be easily reused and updated as needed.

To test:

* Start the simulator.
* Make sure you have the latest mobile secrets for the iOS app.
* Run the Login Tests in the WordPressUITests scheme on both iPad and iPhone.
